dataflux_pytorch/benchmark/checkpointing/multinode/benchmark-deploy.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: jobset.x-k8s.io/v1alpha2
 kind: JobSet
 metadata:
-  name: yashsha-checkpoint
+  name: my-job-run
```

Comment:
I see in other areas it looks like you can use variables such as `$(JOBSET_NAME)`. Would it make sense to use variables instead of placeholders? That would help make it more explicit what values the user need to provide instead of interpreting these values as placeholders.

Reply:
$(JOBSET_NAME) is something which will be replaced by GKE automatically, whereas these are the values which customer needs to change explicitly, but we can do something like `<my_job_run>` since `$()` is reserved for automated replacements.

Reply:
Done